README.md

```diff
@@ -235,10 +235,10 @@ publish.standard();
                 <code>cleanCmd</code>
             </td>
             <td>
-                The CLI to execute which cleans up any temporary changes to the package.json file.
+                The CLI to execute which cleans up any temporary changes to the package.json and package-lock.json files.
                 <ul>
                     <li>
-                        <code>${package}</code> will be substituted with the path to the executing modules package.json file.
+                        <code>${package}</code> will be substituted with the path to the executing modules package.json and package-lock.json files.
```

Comment:
This line seems to have been incorrect and as such still is, because there is not substitution of a `${package}` token. Looking at the publish.clean code, it seems that the command is always executed as is, without interpolating any values. https://github.com/fluid-project/fluid-publish/blob/master/publish.js#L274-L275

Reply:
@jobara  Yes, I also observed it while testing the command. Should I remove this line completely and also edit the corresponding default command to `git checkout -- package.json package-lock.json` in this file?

Or do you need me to make changes so that `cleanCmd` starts accepting values from command line?

Reply:
@atisayajain I think the former. publish.clean will change to the specified moduleRoot first before reverting the package file. https://github.com/fluid-project/fluid-publish/blob/master/publish.js#L260-L279

So for now I think it's okay to leave the functionality as is and just update the documentation. And yes, it looks like you should remove the line and update the default command as you specified.